crates/node/src/constants.rs

```diff
@@ -1,46 +1,2 @@
-pub const SHARINGAN_CHAIN_SPEC_URL: &str = "https://raw.githubusercontent.com/keep-starknet-strange/madara/main/configs/chain-specs/testnet-sharingan-raw.json";
-
-pub const GENESIS_ASSETS_URL: &str =
-    "https://raw.githubusercontent.com/keep-starknet-strange/madara/main/configs/genesis-assets/";
-
-pub const GENESIS_ASSETS_FILES: [&str; 5] =
-    ["Account.json", "AccountBaseImpl.json", "CallAggregator.json", "genesis.json", "Proxy.json"];
-
-pub const CAIRO_CONTRACTS_URL: &str =
-    "https://raw.githubusercontent.com/keep-starknet-strange/madara/main/configs/cairo-contracts/";
-
-pub const CAIRO_CONTRACTS_FILES: [&str; 33] = [
-    "ArgentAccount.json",
-    "bigint.json",
-    "BraavosAccount.json",
-    "calls.json",
-    "constants.json",
-    "Counter.json",
-    "ec.json",
-    "ec_mulmuladd.json",
-    "ec_mulmuladd_secp256r1.json",
-    "emit_multiple_events_across_contracts.json",
-    "emit_single_event.json",
-    "ERC20.json",
-    "ERC721.json",
-    "Example.sierra.json",
-    "field.json",
-    "guards.json",
-    "l1_handler.json",
-    "library.json",
-    "NoValidateAccount.json",
-    "OpenzeppelinAccount.json",
-    "Proxy.json",
-    "security_test.json",
-    "signature.json",
-    "test.json",
-    "UnauthorizedInnerCallAccount.json",
-    "UniversalDeployer.json",
-    "upgradable.json",
-    "cairo_1/erc20.casm.json",
-    "cairo_1/erc20.sierra.json",
-    "cairo_1/HelloStarknet.casm.json",
-    "cairo_1/HelloStarknet.sierra.json",
-    "cairo_1/NoValidateAccount.casm.json",
-    "cairo_1/NoValidateAccount.sierra.json",
-];
+pub const DEFAULT_CONFIGS_URL: &str =
+    "https://raw.githubusercontent.com/d-roak/madara/feat/configs-index/configs/index.json";
```

Comment:
Will it be hosted on you github workspace lastingly?

Reply:
Or is it just for development?

Reply:
for the CI, once we merge I'll do a PR to replace `constants.rs` and `index.json`, to point to the official repo

Reply:
Great